composer.json

```diff
@@ -24,5 +25,19 @@
         "allow-plugins": {
             "php-http/discovery": true
         }
+    },
+    "require-dev": {
+        "phpunit/phpunit": "^9.5",
+        "mockery/mockery": "2.0.x-dev",
+        "open-telemetry/sdk": "^1.0",
+        "open-telemetry/opentelemetry-propagation-traceresponse": "dev-main",
+        "open-telemetry/opentelemetry-propagation-server-timing": "dev-main",
+        "symfony/http-client": "^6.0",
```

Comment:
Why `open-telemetry/sdk` and `symfony/http-client` are in dev, they are required for package to work